### PR TITLE
TypifiedElement implement WebElement

### LINKED ISSUE #120: TypifiedElement implementing WebElement?
Is there a reason that `TypifiedElement` doesn't implement `WebElement`?

The main reason for asking is to simplify use of `ExpectedConditions`, especially when using a list of them e.g. `List<CheckBox> checkboxes`.

I'd like to do: `wait.until(ExpectedConditions.visibilityOfAllElements(checkboxes))` but because visibilityOfAllElements only accepts `List<WebElement>` this doesn't work, equally for `visibilityOf` having to add `.getWrappedElement()` isn't ideal.

Also, as the PageFactory replaces them with lazy proxies if trying to convert that into a list of WebElements to wait for, I'm not sure the following would work?

```
checkboxes.stream()
          .map(WrappedElement::getWrappedElement)
          .collect(toList());
```

Or is there something I've missed?

### PR DESCRIPTION
**Pull request by ham1**
- Added `implement WebElement` to `TypifiedElement`
- Removed methods in sub-classes of `TypifiedElement` which were identical to those newly added in `TypifiedElement`
- Bit of a tidy up of `Form`

Resolves #120

### CODE REVIEW

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java [hunk 1]
```diff
@@ -3,12 +3,13 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```
Comment (artkoshelev) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/Form.java]: please don't use wildcard imports

**Comment by artkoshelev** [comment]
LGTM, just need to remove wildcard imports

**Comment by ham1** [comment]
Fixed, and squashed commits.

**Comment by artkoshelev** [comment]
Same in `TypifiedElement.java` and `Radio.java`

**Comment by ham1** [comment]
Sorry, done now.

**Comment by artkoshelev** [comment]
Thank you, landed!

**Pull request merged**